praw/praw.ini

```diff
@@ -18,3 +18,7 @@ reddit_url=https://www.reddit.com
 
 # The URL prefix for short URLs.
 short_url=https://redd.it
+
+[INTERPOLATION]
```

Comment:
Using a different section here doesn't really make sense as each section defines a site. What's the use-case?

Reply:
right, I didn't realized that was the global ini file; I've moved it to a fixture on the tests directory now.